src/olympia/landfill/serializers.py

```diff
@@ -173,6 +173,53 @@ def create_featured_addon_with_version(self):
             'Created addon {0} for testing successfully'
             .format(addon.name))
 
+    def create_featured_android_addon(self):
+        """Creates a custom addon named 'Ui-Addon-Android'.
+
+        This addon will be a featured addon and will have a featured collecton
+        attatched to it. It will belong to the user uitest.
+
+        It has 1 preview, 2 reviews, and 1 authors.
+
+        It is an Android addon.
+
+        """
+        default_icons = [x[0] for x in icons() if x[0].startswith('icon/')]
+        addon = addon_factory(
+            status=STATUS_PUBLIC,
+            type=ADDON_EXTENSION,
```

Comment:
nit: some of these are defaults of `addon_factory` anyway (`status` and `type`, for example) so you don't need to specify again here.